webapp/playbooks/src/components/actions_modal_action_children.tsx

```diff
@@ -49,7 +47,6 @@ interface OptionType {
 
 export const RunPlaybookChildren = ({playbookId, onUpdate, editable}: RunPlaybookProps) => {
     const {formatMessage} = useIntl();
-    const portalEl = useFloatingPortalNode();
```

Comment:
Is there by chance a weird side effect of calling `useFloatingPortalNode` that this code relies upon? (cc @calebroseland)

Reply:
Should be safe to remove. Shouldn't have any dependencies on useFloatingPortalNode side effects. It explicitly depends on our own modal infra (`#root-poral`—this is static in the app and always there).